sycl/doc/EnvironmentVariables.md

```diff
@@ -195,8 +195,7 @@ variables in production code.</span>
 | `SYCL_QUEUE_THREAD_POOL_SIZE` | Positive integer | Number of threads in thread pool of queue. |
 | `SYCL_DEVICELIB_NO_FALLBACK` | Any(\*) | Disable loading and linking of device library images |
 | `SYCL_PRINT_EXECUTION_GRAPH` | Described [below](#sycl_print_execution_graph-options) | Print execution graph to DOT text file. |
-| `SYCL_DISABLE_EXECUTION_GRAPH_CLEANUP` | Any(\*) | Disable cleanup of finished command nodes at host-device synchronization points. |
-| `SYCL_DISABLE_POST_ENQUEUE_CLEANUP` | Any(\*) | Disable cleanup of enqueued command nodes during submission. |
```

Comment:
Would it make sense to keep it here and specify that it is deprecated until we also remove the behavior for good?

Reply:
I don't have a strong preference here, but since we do list at least some of the deprecated variables in this doc, I added it back.